.github/workflows/veracodescan.yml

```diff
@@ -0,0 +1,41 @@
+name: Push package for veracode scan
+
+on: 
+  pull_request:
+    types: [closed]
+    branches:
+      - master
+
+jobs:
+  pre:
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.skip-check.outputs.should_skip }}
+    steps:
+    - id: skip-check
+      uses: fkirc/skip-duplicate-actions@master
+      with:
+        github_token: ${{ github.token }}
+        paths_ignore: '["**.md", "dev/**"]'
+  veracode-scan:
+    name: veracode scan
+    runs-on: ubuntu-latest
+    needs: pre
+    steps:
+      - name: Download pull request artifact
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: test.yml
+          pr: ${{github.event.pull_request.number}}
+          name: dist
+      - name: Upload and scan
+        uses: veracode/veracode-uploadandscan-action@master
+        with:
+          filepath: 'cf-mendix-buildpack.zip'
```

Comment:
Will this work? The path is (correct me if I'm wrong) `dist/cf-mendix-buildpack.zip`

Reply:
@markbastiaans I have tested with the above path _{filepath: 'cf-mendix-buildpack.zip'}_ and it was able to upload the package successfully. 
Earlier when I was using the **run** command to do the veracode scan, i tried using dist/cf-mendix-buildpack.zip but it gave me an error saying package not found hence I changed it to above one